test_account_update_with_custom_creds_using_cli - remove openshift-storage namespace dependency

Prevent issue:

```
2024-11-26 12:28:01  FAILED
2024-11-26 12:28:01  __ TestCustomCredsUsingMcgCli.test_account_update_with_custom_creds_using_cli __
2024-11-26 12:28:01  
2024-11-26 12:28:01  self = <mcg.test_custom_credentials_using_mcg_cli.TestCustomCredsUsingMcgCli object at 0x7f4bcb792d00>
2024-11-26 12:28:01  mcg_account_factory = <function mcg_account_factory_fixture.<locals>.mcg_account_factory_implementation at 0x7f4bc7f23670>
2024-11-26 12:28:01  
2024-11-26 12:28:01      @skipif_ocs_version("<4.17")
2024-11-26 12:28:01      def test_account_update_with_custom_creds_using_cli(self, mcg_account_factory):
2024-11-26 12:28:01          """
2024-11-26 12:28:01          1. Create a new account using MCG CLI
2024-11-26 12:28:01          2. Retrive creds of the account
2024-11-26 12:28:01          3. Change creds of the with custom value suing CLI
2024-11-26 12:28:01          4. Verify creds are updated using CLI
2024-11-26 12:28:01          5. Validate custom credentials against against length and valid characters
2024-11-26 12:28:01          """
2024-11-26 12:28:01          if not Path(mcg_cli).exists():
2024-11-26 12:28:01              retrieve_cli_binary(cli_type="mcg")
2024-11-26 12:28:01      
2024-11-26 12:28:01          output = run_cmd(cmd=f"{mcg_cli} version")
2024-11-26 12:28:01          logger.info(output)
2024-11-26 12:28:01          account_name = get_random_str(5)
2024-11-26 12:28:01          original_acc_credentials = mcg_account_factory(name=account_name)
2024-11-26 12:28:01          original_secret_key = original_acc_credentials["access_key"]
2024-11-26 12:28:01          original_access_key = original_acc_credentials["access_key_id"]
2024-11-26 12:28:01      
2024-11-26 12:28:01          logger.info(f"Original access key: {original_access_key}")
2024-11-26 12:28:01          logger.info(f"Original secret key: {original_secret_key}")
2024-11-26 12:28:01          logger.info("Updating noobaa account credentials with custom values")
2024-11-26 12:28:01          new_access_key = get_random_str(20)
2024-11-26 12:28:01          new_secret_key = get_random_str(40)
2024-11-26 12:28:01          self.update_nb_account(account_name, new_access_key, new_secret_key)
2024-11-26 12:28:01          retrived_access_key, retrived_secret_key = get_s3_credentials_from_secret(
2024-11-26 12:28:01              f"noobaa-account-{account_name}"
2024-11-26 12:28:01          )
2024-11-26 12:28:01          logger.info(f"retrived access key: {retrived_access_key}")
2024-11-26 12:28:01          logger.info(f"retrived secret key: {retrived_secret_key}")
2024-11-26 12:28:01  >       assert (
2024-11-26 12:28:01              retrived_access_key == new_access_key
2024-11-26 12:28:01          ), "Mismatch in updated and new access key"
2024-11-26 12:28:01  E       AssertionError: Mismatch in updated and new access key
2024-11-26 12:28:01  E       assert '2xmlmPugYbnLh4PqIQyZ' == '0ths855l0pulp3bqox9g'
2024-11-26 12:28:01  E         - 0ths855l0pulp3bqox9g
2024-11-26 12:28:01  E         + 2xmlmPugYbnLh4PqIQyZ
2024-11-26 12:28:01  
2024-11-26 12:28:01  tests/functional/object/mcg/test_custom_credentials_using_mcg_cli.py:69: AssertionError
```

namespace is dynamic, we don’t use “openshift-storage” hardcoded namespace, instead using default value from the config when needed. Whatever deployment is used it will pick either “openshift-storage” or  any other set at ENV_DATA. No dependency on deployment and namespace.
So in code we'll have to replace it with config.ENV_DATA["cluster_namespace"]

job link - https://url.corp.redhat.com/f37e403

--- a/tests/functional/object/mcg/test_custom_credentials_using_mcg_cli.py
+++ b/tests/functional/object/mcg/test_custom_credentials_using_mcg_cli.py
@@ -9,6 +9,7 @@ from ocs_ci.framework.testlib import (
     red_squad,
     mcg,
 )
+from ocs_ci.framework import config
 from ocs_ci.ocs import constants
 from ocs_ci.helpers.helpers import retrieve_cli_binary
 from ocs_ci.utility.utils import run_cmd, get_random_str
@@ -27,11 +28,12 @@ class TestCustomCredsUsingMcgCli(MCGTest):
         """
         Update noobaa account with custom credential values
         """
+        namespace = config.ENV_DATA["cluster_namespace"]
         output = run_cmd(
             cmd=f"{mcg_cli} account credentials {account_name} "
             + f"--access-key={access_key} "
             + f"--secret-key={secret_key} "
-            + "-n openshift-storage",
+            + f"-n {namespace}",
             ignore_error=True,
         )
         logger.info(output)

